socorro/processor/rules/breakpad.py

```diff
@@ -303,35 +278,34 @@ def __repr__(self):
 
     def predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
         if (
-            processed_crash.product != "Firefox"
-            or not processed_crash.os_name.startswith("Windows")
-            or processed_crash.cpu_arch != "x86"
+            processed_crash.get("product", "") != "Firefox"
+            or not processed_crash.get("os_name", "").startswith("Windows")
+            or processed_crash.get("cpu_arch", "") != "x86"
```

Comment:
I'll add a note to myself to ask Gabriele and find out if this is still true. It might not be.